lib/Backend/GlobOpt.cpp

```diff
@@ -19058,6 +19058,44 @@ GlobOpt::OptDstIsInvariant(IR::RegOpnd *dst)
 }
 
 void
+GlobOpt::OptHoistToLandingPadUpdateValueType(
+    BasicBlock* landingPad,
+    IR::Instr* instr,
+    IR::Opnd* opnd,
+    Value* opndVal)
+{
+    if (instr->m_opcode == Js::OpCode::FromVar)
```

Comment:
Is the reason for not updating the value types in case of a FromVar that there is FromVar specific code in OptHoistInvariant that needs to know if the value types were different? If so, would you still need this FromVar check if you call this method after adding the bailout to FromVar?